src/components/Task/Task.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./Task.css";
+import { useDrag } from "react-dnd";
+
+const Task = ({ task }) => {
+    const [{ isDragging }, drag] = useDrag(() => ({
```

Comment:
can you move to separate handle function

Reply:
done